blog/2020-07-31-temporal-v0.28.0-changelog.md

```diff
@@ -0,0 +1,111 @@
+---
+tags:
+- changelog
+- temporal
+- v1
+posted_on_: 2020-07-31T00:00:00
+id: temporal-v0.28.0-changelog
+title: Temporal v0.28.0 changelog
+author: Cully Wakelin
+author_title: SDE Technical Writer
+author_image_url: https://avatars2.githubusercontent.com/u/34380806?s=400&u=5cd38b5e4416a5d10cdf9ebd386eec1d02f0b067&v=4
+release_version: v0.28.0
+---
+
+<!--truncate-->
+
+Today is an exciting day for us! Temporal is releasing its newest version, v0.28.0, bringing us one step closer to a stable production ready release. We thank all of the users who have helped us iterate and move the product forward.
+
+This changelog includes all of the changes since Temporal forked from Cadence, and should help users who are interested in upgrading.
+
+## What has changed?
+
+- **Cadence Oct 2019** → **Temporal v0.28.0 Alpha**: The software version has bumped.
+- **Thrift/TChannel** → **Protobug/gRPC**:
+    - Server Thrift API → Protobuf based Go API (go.temporal.io/api).
+    - All communication over the wire can be encrypted via [TLS](https://docs.temporal.io/docs/configuring-temporal-server#tls). Note: Support for TLS in `tctl` and `temporal-web` is still in-progress.
+    - Thrift errors have been converted to custom service errors backed by standard [gRPC error codes](https://pkg.go.dev/google.golang.org/grpc/codes).
+    - Standard gRPC health service is used for health checks.
+    - Removed Uber YARPC library.
+- **Helm charts** ☸️  : [Deploy Temporal server to a Kubernetes Cluster](https://github.com/temporalio/helm-charts).
+- **Entity renames** 🏷️ :
+    - `Decision` → `Command`
+    - `DecisionTask` → `WorkflowTask`
+    - `Decider` → `Workflow`
+    - `Domain` → `Namespace`
+    - `NamespaceStatus` → `NamespaceState`
+    - `TaskList` → `TaskQueue`
+    - `ArchivalStatus` → `ArchivalState`
+- **Import path rename** 📦 : go.uber.org/cadence → go.temporal.io/sdk
+- **Workflow timeout refactors** ⌛:
+    - Workflow timeouts are optional.
+    - Workflow Executions have 3 timeouts available:
+        - Renamed:`ExecutionStartToCloseTimeout` → `WorkflowRunTimeout`.
+        - `WorkflowExecutionTimeout` is new. It limits the total Workflow execution time including retries and continue-as-new. It also enables limiting the total time of a cron job. The default is 10 years. `WorkflowRunTimeout` defaults to the value of `WorkflowExecutionTimeout`.
+        - Renamed:`DecisionTaskStartToCloseTimeout` → `WorkflowTaskTimeout`. Defaults to 10 seconds.
+- **Workflow retry refactors** 🔁 :
+    - `ExpirationInterval` has been removed from `RetryPolicy`. Instead, `WorkflowExecutionTimeout` and `ScheduleToClose` are used to stop retries for Workflows and Activities respectively.
+- **Activity retry refactors** 🔁 :
+    - Temporal includes default `RetryOptions` for Activities.
+    - To disable Activity retries, configure Activities with `RetryOptions` that set `maximumAttempts` to 1.
+- **tctl (CLI) updates** 🧰 :
+    - New admin commands:
```

Comment:
Maybe prepend the sample commands with admin where appropriate? "admin shard describe" etc?